setup.py

```diff
@@ -36,7 +36,7 @@
     "pymongo[tls,srv]~=3.8",
     "numpy~=1.16",
     "scipy~=1.2",
-    "tensorflow~=1.14.0",
+    "tensorflow~=1.15.0",
```

Comment:
@wochinge do you know if tensorflow does not follow the "if >1.0, pin the minor, not the patch" rule? if not i would rather pin `~=1.14`

Reply:
No, I also think Tensorflow should be an exception here 👍🏼 Pin ~=1.15

Reply:
If it is an exception, it would be ~=1.15.0, not ~=1.15, correct?

Reply:
yes 🙈

Reply:
~=1.15.0 it is